Sources/ChatToys/Utils/WebContext.swift

```diff
@@ -52,7 +56,7 @@ public struct WebContext: Equatable, Codable {
             } else {
                 lines.append("<webpage domain='\(page.searchResult.url.hostWithoutWWW)'>")
             }
-            lines.append(page.markdownWithSnippetAndTitle)
+            lines.append(page.markdownWithNodeIds)
```

Comment:
Can this be optional? E.g. can we turn asXML into a function and have nodeIds as an option? (And then keep the existing asXML method for backwards compatibility?)